### PR TITLE
Option to show name of the LSP in code actions

### PR DESCRIPTION
**Pull request by MaximilianLloyd**
This is useful if you have many active LSPs. Won't be useful for everyone though which is why i've set it to be disabled by default.

### CODE REVIEW

File: doc/lspsaga.nvim.txt [hunk 1]
```diff
@@ -301,6 +301,7 @@ Default options:
 >
       code_action = {
         num_shortcut = true,
+	show_server_name = false,
```
Comment (glepnir) [inline doc/lspsaga.nvim.txt:304]: don't edit this file . it generate by ci

File: lua/lspsaga/codeaction.lua [hunk 2]
```diff
@@ -148,8 +151,9 @@ function act:send_code_action_request(main_buf, options, cb)
     self.action_tuples = {}
 
     for client_id, result in pairs(results) do
+			local name = vim.lsp.get_client_by_id(client_id).name
```
Comment (glepnir) [inline lua/lspsaga/codeaction.lua:154]: these code in wrong pos. you should get the name when the option equals to true.

**Review by glepnir — CHANGES_REQUESTED** [review]
logic is not correct. tuples still 2 . just get client name in option check then set to title.

**Comment by MaximilianLloyd** [comment]
> 

Good point :smile: . Fixed now.

**Comment by MaximilianLloyd** [comment]
Sorry about the wrong formatting. I ran stylua now to fix it.

**Pull request merged**